src/demo/level.rs

```diff
@@ -3,14 +3,38 @@
 use bevy::prelude::*;
 
 use crate::{
+    asset_tracking::LoadResource,
+    audio::music,
     demo::player::{PlayerAssets, player},
     screens::Screen,
 };
 
+pub(super) fn plugin(app: &mut App) {
+    app.register_type::<LevelAssets>();
+    app.load_resource::<LevelAssets>();
+}
+
+#[derive(Resource, Asset, Clone, Reflect)]
+#[reflect(Resource)]
+pub struct LevelAssets {
```

Comment:
needs to be `pub` because `spawn_level` uses it, which itself is used across modules.